packages/frontend-v2/components/Button/FormButtons.tsx

```diff
@@ -0,0 +1,35 @@
+import { Button, ButtonProps, ComponentWithAs } from "@chakra-ui/react";
+
+export const SubmitButton: ComponentWithAs<"button", ButtonProps> = ({
+  children,
```

Comment:
Is there a reason for not using `forwardRef` here but use it for the `StringInput`?

Reply:
I think its general consensus to avoid using forward refs whenever possible, cuz this is just a component that calls a parent's function